src/pages/workspace/bills/WorkspaceBillsFirstSection.js

```diff
@@ -54,11 +55,11 @@ const WorkspaceBillsFirstSection = (props) => {
                 <Text>
                     {props.translate('workspace.bills.askYourVendorsBeforeEmail')}
                     {props.user.isFromPublicDomain ? (
-                        <TouchableOpacity
-                            onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
+                        <TextLink
+                            href="https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/"
```

Comment:
Using `href` here is not correct. It will result in calling the `Link.openUrl` on native. but we want it to be opened via `Link.openExternalLink(`. IMO, `onPress` handler will be best suited.